runtimes/eden/src/pallets_util.rs

```diff
@@ -153,12 +153,21 @@ impl pallet_uniques::Config for Runtime {
 pub enum SponsorshipType {
 	AnySafe,
 	Uniques,
+	// Nft,
```

Comment:
I don't see why we need a AnySafe

Reply:
We need to provide the option to allow any function that may find a use case to be sponsored. We'd just need to exempt unsafe (in terms of being able to violate the sponsorship contract), hence the name AnySafe.